apps/web/src/components/Filters/RollupFilter.tsx

```diff
@@ -35,12 +29,13 @@ export const ROLLUP_OPTIONS: Option[] = [
       prefix: <RollupIcon rollup={rollupName.toLowerCase() as Rollup} />,
       label: capitalize(rollupName),
     };
-  }),
-];
+  }
+);
 
 export const RollupFilter: FC<RollupFilterProps> = function ({
   onChange,
   selected,
+  isDisabled,
```

Comment:
This is a bit of a nitpick, but can we maintain consistency with other HTML element props and use `disabled` instead of `isDisabled`?

Reply:
Done 👍